inc/sairoute.h

```diff
@@ -142,35 +144,42 @@ typedef sai_status_t (*sai_create_route_fn)(
  *
  * Note: IP prefix/mask expected in Network Byte Order.
  *
+ * @param[in] switch_id - Switch id
  * @param[in] route_entry - route entry
  *
  * @return #SAI_STATUS_SUCCESS on success Failure status code on error
  */
 typedef sai_status_t (*sai_remove_route_fn)(
-        _In_ const sai_route_entry_t *route_entry);
+        _In_ sai_switch_id_t switch_id,
+        _In_ const sai_route_entry_t *route_entry, 
```

Comment:
while I agree the switch be every where, I think it is better to put the switch_id into the sai_route_entry_t, sai_neighbor_entry_t, and sai_fdb_entry_t. 

Because we only agree to add switch_id as a parameter in the create API, not in set,get,remove API.